scss/variables/_spacing.scss

```diff
@@ -1,32 +1,18 @@
 // Spacing
 $spacing-scale: $base-scale * 1;
 
-$spacing-0: 0;
```

Comment:
Why are these variables removed? Won't this break stylesheets where camp is being imported as `.scss`?

Reply:
Ah - I was too hasty with this one. This was the only partial within the camp-css repository that was using those particular variables so it seemed superfluous. I didn't consider the affect on anything outside the repo, though. Adding back in...